README.md

```diff
@@ -1,34 +1,33 @@
+<p align="center">
+  <img src="book/src/powdr_wires.png" width="600">
+</p>
+
 # powdr
 
 [![Matrix Chat](https://img.shields.io/badge/Matrix%20-chat-brightgreen?style=plastic&logo=matrix)](https://matrix.to/#/#powdr:matrix.org)
 [![Twitter Follow](https://img.shields.io/twitter/follow/powdr_labs?style=plastic&logo=twitter)](https://twitter.com/powdr_labs)<!-- markdown-link-check-disable-line -->
 
-*powdr* is an extended polynomial identity (PIL) and zk-focused assembly (zkASM)
-language written in Rust, focused on modularity and excellent developer experience.
-
-WARNING: This is a proof-of-concept implementation. It is missing many internal checks. DO NOT USE FOR PRODUCTION!
+> WARNING: This codebase is experimental and has not been audited. DO NOT USE FOR PRODUCTION!
 
-## Basic Concept
+For detailed documentation please visit [the powdr book](https://docs.powdr.org/).
 
 *powdr* is a toolkit that helps build zkVMs and similar proof frameworks.
 
 It has two main components:
 
-- A polynomial identity language that allows you to define polynomial constraints, lookups, etc.
-- An extensible assembly language to perform dynamic executions.
+- powdr-asm: an extensible assembly IR language to perform dynamic executions.
```

Comment:
would the word `self-defining` be fitting?

Reply:
where exactly?

If for the language itself I don't really like it. The language exists with very few builtins, but they exist: register datatypes, register assignments, machines, inline constraints, instructions, labels.

How about:

- powdr-asm: an extensible assembly IR language to define any VM / instruction set

?

Reply:
I'm mainly thinking about the fact that the language is not just `extensible`, but much more. But I'm also fine with it as is.

Reply:
ok, this sentence is mostly the same, we can re-visit in the future